src/autoscaling.rs

```diff
@@ -52,10 +55,22 @@ pub async fn matcher(opt: AutoScalingGroupOpt) {
 async fn info(opt: SearchInfoQueryOpt) {
     let asg = get_auto_scaling_groups(&SearchQueryOpt { query: opt.query }).await;
     let len = asg.len();
-    let tag_column: Vec<String> = opt
-        .tag_columns
-        .map(|t| split(&*t, true))
-        .unwrap_or_default();
+
+    let tag_column: Vec<String> = if opt.show_all_tags {
+        asg.iter()
+            .map(|t| t.tags.iter().map(|ot| ot.key.to_string()).collect())
+            .collect::<Vec<Vec<String>>>()
+            .into_iter()
+            .flatten()
+            .collect::<Vec<String>>()
+            .into_iter()
+            .unique()
+            .collect()
```

Comment:
Do not need use `collect` in method chains, use collect at last like below.
```rust
.map(|t| t.tags.iter().map(|ot| ot.key.to_string()))
            .flatten()
            .unique()
            .collect()
```

Reply:
Oops, I didn't know that.
Thanks!

Reply:
I fixed it e004b19